src/taco-cli/test/buildAndRunTelemetry.ts

```diff
@@ -291,7 +291,12 @@ module BuildAndRunTelemetryTests {
             (typeof telemetryProperties === "undefined").should.be.equal(false);
             validateGzipedSize(telemetryProperties, "ios", iosExpectedGzipedSize);
             validateGzipedSize(telemetryProperties, "android", androidGzipSize);
-            telemetryProperties.should.eql(expected); // We are comparing the objects, after overriding the sizes with the expected values
```

Comment:
Might be able to use `telemetryProperties.should.containEql(expected)`? Looks like `{x: 1, y: 2}.should.containEql({y:2})` works and `{x: 1, y: 2}.should.containEql({z:3})` fails, so I think that's what we want here